human-dynamics-estimator/src/HumanDynamicsEstimator.cpp

```diff
@@ -23,35 +34,132 @@ double HumanDynamicsEstimator::getPeriod()
 //---------------------------------------------------------------------
 bool HumanDynamicsEstimator::configure(yarp::os::ResourceFinder &rf)
 {
-    count = 0;
-    /*
-     * ------Human model loading and initialization
+    
+    /* ------Open a port:i for the human joint configuration
      */
-    modelFilename = rf.findFile("humanModel");
+    if (!m_humanJointConfiguration_port.open("/humanJointConfiguration:i"))
+    {
+        yError() << "Unable to open port /humanJointConfiguration:i";
+        return false;
+    }
+    
+    /* ------Open a port:i for the human forces
+     */    
+    if (!m_humanForces_port.open("/humanForces:i"))
+    {
+        yError() << "Unable to open port /humanForces:i";
+        return false;
+    }    
+    
+    /*
+     * ------Human model loading 
+     */  
+    // TODO: substitution with thrift
+    std::vector<std::string> joints;
```

Comment:
As an offline mode can be useful, you can implement the reading of the joint list in two ways: 
if online, read from the RPC port of the `human-state-provider` module, or if offline, read from configuration file.
See the code of the `human-viz-bridge`